src/Nav.tsx

```diff
@@ -0,0 +1,31 @@
+import { NavLink } from "react-router-dom"
+import classNames from "classnames";
+
+const isActiveTab = ({ isActive }: { isActive: boolean }) => classNames(
+  'navbar-item', { 'is-active': isActive },
+);
+
+export const NavPage = () => {
+return(
+  <nav
+    className="navbar is-light is-fixed-top is-mobile has-shadow"
+    data-cy="Nav"
+  >
+  <div className="container">
+    <div className="navbar-brand">
+      <NavLink
+        to="/"
+        className={isActiveTab}
+      >Home
```

Comment:
```suggestion
      >
        Home
```

Consider fixing indentations accordingly to the code style guide